pkg/util/proxyutil/proxyutil_test.go

```diff
@@ -11,9 +11,9 @@ func TestPrepareProxyRequest(t *testing.T) {
 	t.Run("Prepare proxy request should clear X-Forwarded headers", func(t *testing.T) {
```

Comment:
I am curious about this. Generally proxies append to X-Forwarded-For behavior is to append. Maybe for security reasons we want to clear them? 

So this is changing the behavior (to match what the test says, but I wonder if it says the right thing)?